app/javascript/dashboard/components/CustomBrandPolicyWrapper.vue

```diff
@@ -0,0 +1,28 @@
+<script setup>
```

Comment:
Added this component so that next if there is a custom brand check, we can use the component directly.